src/ken/src/ken_server.erl

```diff
@@ -286,7 +286,9 @@ update_db_indexes(Name, State) ->
                 true -> ok
             end;
         {error, timeout} ->
-            exit(resubmit)
+            exit(resubmit);
+        Else ->
+            exit(Else)
```

Comment:
A missing target is essentially a database_does_not_exist case.

Something is off a missing target is something we should be seeing in the internal replicator not for places like `design_docs()`

Reply:
Revert this back and modified fabric_rpc.erl. Thanks for the review and suggestions.